packages/typespec-ts/test/integration/authApiKey.spec.ts

```diff
@@ -3,20 +3,27 @@ import ApiKeyClientFactory, {
 } from "./generated/authentication/api-key/src/index.js";
 import { assert } from "chai";
 describe("AuthApiKeyClient Rest Client", () => {
-  let client: AuthApiKeyClient;
+  let validKeyClient: AuthApiKeyClient;
+  let invalidKeyClient: AuthApiKeyClient;
 
   beforeEach(() => {
-    client = ApiKeyClientFactory(
+    validKeyClient = ApiKeyClientFactory(
       {
         key: "valid-key"
       },
       { allowInsecureConnection: true }
     );
+    invalidKeyClient = ApiKeyClientFactory(
```

Comment:
is this introduced recently?

Reply:
https://github.com/Azure/cadl-ranch/pull/752/files#diff-7e6ef54edf1535ebe6cbfdf2a7ea48d31f33f797ce5135ec7cb55a616f4f7f1bR29